README.md

```diff
@@ -49,11 +53,13 @@ cd docker-images/
 
 <img width="535" alt="screen shot 2018-10-10 at 12 20 48" src="https://user-images.githubusercontent.com/90316/46729966-22206600-cc87-11e8-9e1a-156d8a6c5e43.png">
 
+---
+
 This will bring up the `stable` versions of all components, referring to their respective `master` branches.
 
-To get the `latest` versions of all components, referring to their `develop` branches, pass the argument `--latest`:
+To run as a publisher, `Brizo` configuration must be set with valid Azure account credentials. This is done in the file [brizo.env](./brizo.env).
```

Comment:
This sentence could replace the "Populate the following..." part under Prerequisites and be removed from down here. Combining the info, like so:

```markdown
To run as a publisher, `Brizo` configuration must be set with valid Azure account credentials. This is done in the file [`brizo.env`](./brizo.env). Follow our tutorial [Set up Azure Storage](https://docs.oceanprotocol.com/tutorials/azure-for-brizo/) to learn how to get those credentials.
```

Then we have said everything in a shorter way and only once.

Reply:
rephrased some parts to reflect this. it's way cleaner now, thanks.